custom/covid/rules/custom_actions.py

```diff
@@ -4,13 +4,41 @@
 
 The following actions can be used in messaging in projects using the ``covid`` custom module.
 """
+from datetime import datetime
+
+from dimagi.utils.parsing import ISO_DATE_FORMAT
+
+from corehq.apps.domain.models import Domain
 from corehq.apps.es.case_search import CaseSearchES
 from corehq.apps.es.cases import case_type
 from corehq.apps.data_interfaces.models import CaseRuleActionResult, AUTO_UPDATE_XMLNS
 from corehq.apps.hqcase.utils import update_case
 from corehq.apps.es import filters
 
 
+def set_all_activity_complete_date_to_today(case, rule):
+    """
+    For any case matching the criteria, set the all_activity_complete_date property
+    to today's date, in YYYY-MM-DD format, based on UTC.
+    """
+    domain_obj = Domain.get_by_name(case.domain)
+    today = datetime.now(domain_obj.get_default_timezone()).strftime(ISO_DATE_FORMAT)
```

Comment:
The docstring says 'based on UTC' but here you're using the timezone of the domain.

Reply:
oops, forgot to update that